.github/workflows/ci.yml

```diff
@@ -28,11 +28,17 @@ jobs:
   run-tests:
     runs-on: [self-hosted, kind-cluster]
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
+    - uses: hashicorp/setup-terraform@v2
+      with:
+        terraform_version: 1.0.11
```

Comment:
nit: would we want to keep the same notation of "latest patch version": `~>1.0`?

Reply:
Sure.  The syntax is a bit different for this config, but I updated it.